proxy/tcpproxy/userspace.go

```diff
@@ -31,6 +37,7 @@ func (r *remote) inactivate() {
 	r.mu.Lock()
 	defer r.mu.Unlock()
 	r.inactive = true
+	plog.Noticef("inactivated %s", r.addr)
```

Comment:
i think this should be warning as i mentioned. also we need to log why it is inactive (the error) and how long will we retry on this remote?